lib/utilities/pipeline.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+var Promise = require('rsvp').Promise;
+var _       = require('lodash-node');
+
+var chalk = require('chalk');
+var blue  = chalk.blue;
+var red   = chalk.red;
+var grey  = chalk.grey;
+
+function Pipeline(hookNames, options) {
```

Comment:
Ok...Will switch it over.

On Thu, 23 Apr 2015 at 07:23 Michael Klein <notifications@github.com> wrote:

> In lib/utilities/pipeline.js
> <https://github.com/ember-cli/ember-cli-deploy/pull/130#discussion_r28938884>
> :
>
> > @@ -0,0 +1,73 @@
> > +'use strict';
> > +
> > +var Promise = require('rsvp').Promise;
> > +var _       = require('lodash-node');
> > +
> > +var chalk = require('chalk');
> > +var blue  = chalk.blue;
> > +var red   = chalk.red;
> > +var grey  = chalk.grey;
> > +
> > +function Pipeline(hookNames, options) {
>
> I think using core object would make it easier for other ember-devs to
> join in. It's not hard with a prototype but when being used to the
> ember.object model I think it's easier to other devs when being able to
> work with a 'normal' object.
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/ember-cli/ember-cli-deploy/pull/130/files#r28938884>.
>